### PR TITLE
[GLUTEN-7493][VL] Update Velox.md to clarify dependency deployment

### LINKED ISSUE #7493: [VL] Dependency deployment guide is outdated
### Description

As mentioned in https://github.com/apache/incubator-gluten/issues/7439#issuecomment-2407492656.

### PR DESCRIPTION
**Pull request by PHILO-HE**
## What changes were proposed in this pull request?

As mentioned in https://github.com/apache/incubator-gluten/issues/7439#issuecomment-2407492656, the guide is outdated.

Fixes #7493.

## How was this patch tested?

N/A

### CODE REVIEW

**Comment by PHILO-HE** [comment]
@ted-jenks, could you help review this pr? Any improvement suggestion is welcome!

**Comment by zhouyuan** [comment]
CC @weiting-chen PTAL

File: docs/get-started/Velox.md [hunk 1]
```diff
@@ -68,27 +68,25 @@ setting up build dependencies.
...
+**Step-by-step build**
 
+Alternative to the above one-step build, you can follow the below guide for step-by-step build.
 Currently, Gluten is using a [forked Velox](https://github.com/oap-project/velox/) which is daily updated based on [upstream Velox](https://github.com/facebookincubator/velox).
 
-Scripts under `/path/to/gluten/ep/build-velox/src` provide `get_velox.sh` and `build_velox.sh` to build Velox separately, you could use these scripts with custom repo/branch/location.
+```bash
 
-Velox provides arrow/parquet lib. Gluten cpp module need a required VELOX_HOME parsed by --velox_home, if you specify custom ep location, make sure these variables be passed correctly.
+./dev/builddeps-veloxbe.sh build_arrow
```
Comment (zhouyuan) [inline docs/get-started/Velox.md:86]: Can you please make a note? 
we need this as we slightly modified arrow, but patches are merged into upstream already, so in the future we could use upstream arrow

**Comment by zhouyuan** [comment]
@PHILO-HE  Can you please create one ISSUE to track this?

File: docs/get-started/Velox.md (hunk 1 above)
Comment (PHILO-HE) [reply docs/get-started/Velox.md:88]: @zhouyuan, just updated. Thanks!

**Review by zhouyuan — APPROVED** [review]
👍

**Pull request merged**